guides/common/modules/con_defining-your-content-library.adoc

```diff
@@ -0,0 +1,19 @@
+[id="defining-your-content-library_{context}"]
+= Definining your content library
+
+To ensure that your {ProjectServer} can manage software and provide it to your hosts, you must enable repositories and synchronize them.
```

Comment:
From a Red Hat content perspective this sentence makes sense, but not for any other content type. In our Katello vocabulary, "enabling" a repository means going to the Red Hat Repositories page and enabling certain RH repos, which auto-creates them as repositories in specific products for the user.

For any other non-RH content, the user has to create a custom product and add custom repositories within.

To make this snippet more neutral, maybe "enable" should be replaced with "create"? And then some reference to enabling RH repositories could be added.

Reply:
What if I take a bit of a bold step and move the contents of [Sources of content](https://docs.theforeman.org/nightly/Planning_for_Project/index-katello.html#content-sources-in-Foreman_planning) to this module instead? I think that would cover this part of the feedback. So, addressed.